resources/js/components/SiteSharingCard.vue

```diff
@@ -0,0 +1,290 @@
+            if (e.response && e.response.status === 403) {
+              this.$alert('You are not authorized to complete this action.')
+            } else if (e.response && e.response.status === 422) {
+              const errors = new Errors(e.response.data)
+              this.$alert(errors.toArray().join(' '))
+            } else {
+              this.$alert('Unable to process your request at this time. Please try refreshing the page or contact us.')
+            }
+            console.error(e)
+          }
+
+          this.loading = false
+        }
+      },
+
+      async loadInvitations() {
+        try {
+          const {data} = await axios.get(`/web/site-invitations`, {
+            params: {
+              site_id: this.site.id,
+            }
+          })
+          this.invitations = data
+        } catch (e) {
+          if (e.response && e.response.status === 403) {
+            this.$alert('You are not authorized to complete this action.')
+          } else if (e.response && e.response.status === 422) {
+            const errors = new Errors(e.response.data)
+            this.$alert(errors.toArray().join(' '))
+          } else {
+            this.$alert('Unable to process your request at this time. Please try refreshing the page or contact us.')
+          }
+          console.error(e)
+        }
+
+        this.loading = false
+      },
+
+      async inviteUser() {
+        try {
+          await this.userForm.post(`/web/sites/${this.site.id}/invitations`)
+          this.$notify({
+            type: 'success',
+            text: 'Invitation sent successfully.',
+          })
+          this.user = null
+          this.loadInvitations()
+        } catch (e) {
+          if (e.response && e.response.status === 403) {
+            this.$alert('You are not authorized to complete this action.')
+          } else if (e.response && e.response.status === 422) {
+            const errors = new Errors(e.response.data)
+            this.$alert(errors.toArray().join(' '))
+          } else {
+            this.$alert('Unable to process your request at this time. Please try refreshing the page or contact us.')
+          }
+          console.error(e)
+        }
+      },
+
+      async toggleEdit(user) {
+        try {
+          const {data} = await axios.put(`/web/user-sites/site/${this.site.id}/user/${user.id}/toggle-edit`)
+          user.can_edit = data.editable
+          this.$notify({
+            type: 'success',
+            text: 'Updated user permissions successfully',
+          })
+        } catch (e) {
+          if (e.response && e.response.status === 403) {
+            this.$notify({
+              text: 'You are not authorized to complete this action.',
+              type: 'error',
+            })
+          } else if (e.response && e.response.status === 422) {
+            const errors = new Errors(e.response.data)
+            this.$alert(errors.toArray().join(' '))
+          } else {
```

Comment:
Since it's not possible to receive a 422 when there is no validation, this bit of error detection is not actually needed.

```suggestion
          } else {
```